parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -268,11 +268,6 @@ configureCNI() {
     retrycmd_if_failure 120 5 25 modprobe br_netfilter || exit $ERR_MODPROBE_FAIL
     echo -n "br_netfilter" > /etc/modules-load.d/br_netfilter.conf
     configureCNIIPTables
-    {{if HasCiliumNetworkPlugin}}
```

Comment:
there is also a check of HasCiliumNetworkPlugin in L393 when we enabling kubelet, shall we remove it as well?

https://github.com/Azure/AgentBaker/pull/575/files#diff-4f9504893023c819ba00993d7db2a80f9883ef64f249e6e61b06d18094f3d425R388

Reply:
also can we remove HasCiliumNetworkPolicy and HasCiliumNetworkPlugin from baker.go, L463- L466?

Reply:
removed them